test/e2e_node/hugepages_test.go

```diff
@@ -225,12 +225,11 @@ var _ = SIGDescribe("HugePages", framework.WithSerial(), feature.HugePages, "[No
 		restartKubelet(true)
 
 		ginkgo.By("verifying that the hugepages-3Mi resource no longer is present")
-		gomega.Eventually(ctx, func() resource.Quantity {
+		gomega.Eventually(ctx, func() bool {
 			node, err = f.ClientSet.CoreV1().Nodes().Get(ctx, framework.TestContext.NodeName, metav1.GetOptions{})
-			framework.ExpectNoError(err, "while getting node status")
-			// abc, error := node.Status.Capacity["hugepages-3Mi"]
-			return node.Status.Capacity["hugepages-3Mi"]
-		}, 30*time.Second, framework.Poll).Should(gomega.BeNil())
+			_, isPresent := node.Status.Capacity["hugepages-3Mi"]
+			return isPresent
+		}, 30*time.Second, framework.Poll).Should(gomega.BeFalse())
```

Comment:
@saschagrunert Please use `gomega.BeFalseBecause` with an error message to show when it fails.
Same as below.

Reply:
:+1: done